src/Card/Card.scss

```diff
@@ -422,6 +422,8 @@ a .pgn__card {
   border-radius: 0 0 .375rem .375rem;
 
   .pgn__card-status__message-content {
+    width: 100%;
```

Comment:
Glad this is a simple fix. FWIW, `flex-grow: 1;` here would also do the trick, and may make a bit more sense given this element doesn't actually take up 100% of the parent's width (i.e., it grows to take up whatever space is remaining after the space taken up by the icon).

Reply:
Yes, I checked this also work fine. Don't know whether there will be difference in some cases? Just curious. I replaced to `flex-grow: 1;`